### DISCUSSION TITLE
iperf3 not working between Windows and Embedded Linux Target

### DISCUSSION THREAD

**Discussion by gurpreetshanky** [esnet/iperf#1245] [Q&A]
# Context

* Version of iperf3:
3.10

* Hardware:
Custom Embedded System running Linux

* Operating system (and distribution, if any):
Yocto Linux and Windows 10

# Bug Report

iperf3 is not able to connect from Windows machine to Linux Embedded System Target. The same Embedded System Target works with another Linux Laptop. ping works fine between windows machine and Linux Embedded System Target. Any pointers how to further debug this? We do see a weird log on server side when we do CTRL+C on client (iperf3:error - unable to receive parameters from client). We already tried binding to port, binding the addresses, forcing the IPV4 on client side with same results.

**Comment by swlars** [comment]
Hi @gurpreetshanky, 

We have some questions about your test setup. 

* Iperf3 does not officially support Windows. What are you using and what version of iperf3 does it use?
* Do you have a screenshot of the output?

Again, iperf3 does not officially support Windows.

Thanks!
